test/cmp_oeis.jl

```diff
@@ -0,0 +1,32 @@
+using OEIS
```

Comment:
Good idea testing with OEIS data - I believe this means you need to include `OEIS` in `test/REQUIRE`.